packages/editor-ui/src/components/layouts/ResourcesListLayout.vue

```diff
@@ -651,7 +703,7 @@ const loadPaginationFromQueryString = async () => {
 }
 
 .search {
-	max-width: 240px;
+	// max-width: 240px;
```

Comment:
intentional, or should be removed completely?

Reply:
Should be removed